src/app/application/components/route-view/RouteView.vue

```diff
@@ -101,6 +100,7 @@ const children: StringNamedRouteRecordRaw[] = (router.getRoutes().find((route) =
   item.name = String(item.name)
   return item as StringNamedRouteRecordRaw
 }) ?? [])
+const active = computed(() => children.find((item) => item.name === route.name || item.meta?.module === route.meta.module))
```

Comment:
**Comment**: Instead of providing `active` as a function determining the active route, I turned it into a computed property. For one, this avoids requiring the consumer of RouteView to also access `useRoute()` (RouteView already accesses it and so it can make that decision completely internally). For another, it avoids recomputations triggered by functions in templates.